### DISCUSSION TITLE
Run in Docker

### DISCUSSION THREAD

**Discussion by sengiv** [gen-so/Genso.Astrology#8] [Show and tell]
![docker-support](https://user-images.githubusercontent.com/43817262/222875869-5b6143cc-73c1-420f-89fe-0acf81715ef5.jpg)

## Get started
Pull the latest image from [DockerHub](https://hub.docker.com/r/vedastro/api)
```powershell
docker pull vedastro/api
```

## Notes
- Available as of v12.16, commit a7869b1557e75fc98c245e72be316d981bfcb16a
- [Demo Test in Visual Studio 2022](https://user-images.githubusercontent.com/43817262/222876851-07ea836e-fdf6-4674-a43e-35dfe0036e41.gif)
- Thanks to @razukc for suggesting this.

**Comment by razukc** [comment]
Glad to help. I tested it and it successfully operated. Thanks.

**Reply by sengiv** [reply]
![image](https://user-images.githubusercontent.com/43817262/227263808-ebe2e559-1277-4340-802f-cfc31b953599.png)
